1001-2000/LIT1180Baenta.xml

```diff
@@ -57,14 +57,57 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
          <change who="DR" when="2018-04-24">Added xml:lang to title</change>
          <change who="DR" when="2018-04-25">Corrected transliteration</change>
          <change when="2019-11-13" who="ES">added incipit</change>
+         <change when="2019-11-20" who="ES">added colophon and relations</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <div type="edition" xml:lang="gez">
             <div type="textpart" subtype="incipit"><ab>በስመ፡ … ድርሳን፡ ዘደረሰ፡ አባ፡ ያዕቆብ፡ ኢጲስ፡ ቆጶስ፡ ዘሀገረ፡ ሥሩግ፡ በእንተ፡ አብርሃም፡ አቡነ፡ አርከ፡ እግዚአብሔር፡</ab></div>
+            <div type="textpart" subtype="subscriptio explicit" xml:id="colophon"><ab>
+               ወይረሲ፡ እግዚአብሔር፡ ማኅደሮ፡ ለአቡነ፡ አባ፡ ሰላማ፡ ጳጳስነ፡ ምስሌሆሙ፡ ደርገ፡ አሜን፨ 
+               ለዝንቱ፡ ድርሳን፡ ዘአጽሐፎ፡ አቡነ፡ አባ፡
+               ሰላማ፡ ጳጳስ፡ ርቱዓ፡ ሃይማኖት፡ እምዐረቢ፡ መጽሐፈ፡ ዚአሁ፡ ለግዕዝ፤ ይተሉ፡ ዝየ፡ ተጽሕፎ፡ ዘይቤ፡ አባ፡
+               አትናስዮስ። <note>quoted after GBI-002</note> <note>Subscriptio ascribing the translation
```

Comment:
I think it would be preferable to refer to the manuscript with a `<ref>`, otherwise maybe only those who know the project can identify it.

Reply:
ok